forge-gui/res/cardsfolder/upcoming/effluence_devourer.txt

```diff
@@ -0,0 +1,13 @@
+Name:Effluence Devourer
+ManaCost:B R G
+Types:Creature Crocodile
+PT:4/3
+K:Blitz:B R G
+T:Mode$ Sacrificed | ValidCard$ Creature | ValidPlayer$ You | Execute$ TrigEffect | TriggerZones$ Battlefield | RememberSacrificed$ True | TriggerDescription$ Whenever you sacrifice CARDNAME or another creature, it perpetually gains "{2}, Exile this card from your graveyard: Create an X/X green Ooze creature token, where X is this card's power. Activate only as a sorcery."
```

Comment:
RememberSacrificed$ param is non-functional here
Forge is set up to track the cards that make the trigger fire, though sometimes figuring out exactly what that card is then called for different triggers is a bit gnarly